docs/coordinates/index.rst

```diff
@@ -282,7 +282,7 @@ names, and etc.:
     `~astropy.coordinates.SkyCoord.from_name`,
     `~astropy.coordinates.EarthLocation.of_site`, and
     `~astropy.coordinates.EarthLocation.of_address` are for convenience, and
-    hence are by design rather simple. If you need precise coordinates for an
+    hence are by design rather imprecise. If you need precise coordinates for an
```

Comment:
```suggestion
    hence are by design relatively low precision. If you need more precise coordinates for an
```
that might not be the best either, but in my head at least, "rather imprecise" sounds like "bad and shouldn't be used", whereas I think here we really mean "probably good enough for most cases, but doesn't provide complete and total control"?